calico/felix/config.py

```diff
@@ -163,6 +166,14 @@ def validate_cfg(self):
         self.validate_addr("PluginAddress", self.PLUGIN_ADDR)
         self.validate_addr("ACLAddress", self.ACL_ADDR)
 
+        #*********************************************************************#
+        #* Bind address must be * or an IP address.                          *#
+        #*********************************************************************#
+        if (self.BIND_ADDR != "*" and
+                not common.validate_ipv4_addr(self.BIND_ADDR)):
```

Comment:
Actually, I think this can be a hostname. I think `localhost` is a perfectly valid string to place here.